### ISSUE TITLE
Gnome Keyring Daemon not executable with nfs-mount

### ISSUE THREAD

**Issue by fliwatuet** [ltsp/ltsp#115]
We are using LTSP in our office with home-directories mounted via nfs, Gnome keyring isn't working, after reading https://github.com/ltsp/ltsp/issues/22 I checked if gnome-keyring-daemon is executable on the client and it isn't, though according to the description there it should only be set unexecutable if sshfs is used.

**Comment by alkisg** [comment]
Please login to a client and then upload the output of:

```shell
grep home /proc/mounts
```

**Comment by fliwatuet** [comment]
Here is the output of grep home /proc/mounts

server:/home /home nfs4 rw,relatime,vers=4.2,rsize=1048576,wsize=1048576,namlen=255,hard,proto=tcp,timeo=600,retrans=2,sec=sys,clientaddr=192.168.1.45,local_lock=none,addr=192.168.1.35 0 0

**Comment by alkisg** [comment]
Hmm the current code needs this in ltsp.conf:

```
[clients]
SSHFS=0
```

I'll leave this issue open until I have time to see if we should try to autodetect that, or fix the documentation. Thanks!

**Comment by fliwatuet** [comment]
I added the line to the clients section of my ltsp.conf and ran ltsp initrd after that, but still gnome-keyring-daemon is not executable on the clients after rebooting them.

**Comment by alkisg** [comment]
Ah sorry, I was thinking that pamltsp might be useful outside LTSP, so I didn't make it an LTSP script, so it can't read ltsp.conf parameters.

Try this instead:

```shell
[clients]
POST_INIT_FIX_PAMLTSP="sed 's/gnome-keyring-daemon/unused/g' -i /usr/share/ltsp/client/login/pamltsp"
```

**Comment by fliwatuet** [comment]
That workaround works, thanks a lot for the quick help!

**Comment by alkisg** [comment]
Fix committed in 64ed9137532a707e9a00e3535c9637dbd24abf44.
